Rakefile

```diff
@@ -4,3 +4,5 @@ require "rspec/core/rake_task"
 RSpec::Core::RakeTask.new(:spec)
 
 task :default => :spec
+
+load 'neo4j/rake_tasks/neo4j_server.rake'
```

Comment:
rake neo4j:install to install a DB under db/
rake neo4j:start to start up the DB
rake neo4j:stop to stop the DB